frontend/src/app/app.module.ts

```diff
@@ -13,55 +13,65 @@ See the License for the specific language governing permissions and
 limitations under the License. */
 
 // Angular Imports.
+import { LayoutModule } from '@angular/cdk/layout';
```

Comment:
Alphabetize this section

Reply:
Vishal told me to alphabetize by the from location in the previous pull request

Reply:
We need to be consistent throughout project. We can use the imported module name or the location to sort. Depends on what is easier to process while reading!!!

Reply:
I would think it would be easier to read with the module names alphabetized, so I'll change it to that throughout the project?

Reply:
I was going by the Python style guide, which the equivalent formatting would have you alphabetize by module name. However it looks like the JavaScript style guide actually says "You are advised not to comment on the following issues in code reviews: 1. Import order. Just use the tools"

Unfortunately the tools are not available to you, so we should just pick one and be consistent. I'm fine with either one.

Reply:
I'll just pick the alphabetize by location then, since the project is already following that